docs/src/main/asciidoc/datastore.adoc

```diff
@@ -809,6 +815,14 @@ Delete queries can have the following return types:
 * A collection of entities that were deleted
 * 'void'
 
+Methods can have `org.springframework.data.domain.Pageable` parameter to control pagination and sorting,
+or `org.springframework.data.domain.Sort` parameter to control sorting only.
+See https://docs.spring.io/spring-data/data-commons/docs/current/reference/html/#repositories.query-methods[Spring Data documentation] for details.
+
+For returning multiple items in a repository method, we support Java collections as well as `org.springframework.data.domain.Page` and `org.springframework.data.domain.Slice`.
+If a method's return type is `org.springframework.data.domain.Page`, the returned object will include current page,
```

Comment:
Please put a warning here about using Page vs Slice, that Page requires retrieval of the keys for all entities of the kind and that Slice is much more efficient.